adcm_aio_client/core/objects/cm.py

```diff
@@ -480,15 +486,27 @@ def object(self: Self) -> Object:
     def action(self: Self) -> Action:
         return self._construct(what=Action, from_data=self._data["action"])
 
-    async def wait(self: Self, status_predicate: Callable[[], bool], timeout: int = 30, poll: int = 5) -> None:
-        if self._data["status"] not in (JobStatus.RUNNING, JobStatus.CREATED):
-            return
+    async def wait(
+        self: Self, timeout: int | None, poll_interval: int = 10, exit_condition: Optional[Callable[[], bool]] = None
```

Comment:
Make `self._default_exit_condition` public function not bound to this object and set it as default value for `exit_condition` argument instead of `None`.

It's signature should be `Callable[[Job], bool]`, because otherwise users won't be able to specify their own conditions without bond to an actual job.